### ISSUE TITLE
Mist won't work after updating to 0.5.2

### ISSUE THREAD

**Issue by Wat3v4** [ethereum/go-ethereum#2336]
Everything was working fine.. I've been using since beta 5 and with every update i get the latest one.. 
But after installing the beta 10, It get into mist and if i exit it I'd have to restart my macbook pro to get mist to work again, until i quit it again..
I use firewall on. 
Macbook pro, Yosemite OSX. 

This is the message i got to post on Issues.

Node type: geth
Network: main
Platform: darwin

...Users/JZ/Library/Ethereum/chaindata
I0314 17:39:23.429531    1198 database.go:71] Alloted 78MB cache to /Users/JZ/Library/Ethereum/dapp
I0314 17:39:23.433577    1198 backend.go:314] Protocol Versions: [63 62 61], Network Id: 1
I0314 17:39:23.434027    1198 backend.go:362] Blockchain DB Version: 3
I0314 17:39:23.434999    1198 blockchain.go:214] Last header: #1149063 [b75505c6…] TD=9403514727179431711
I0314 17:39:23.435022    1198 blockchain.go:215] Last block: #1149063 [b75505c6…] TD=9403514727179431711
I0314 17:39:23.435032    1198 blockchain.go:216] Fast block: #1149063 [b75505c6…] TD=9403514727179431711
I0314 17:39:23.435262    1198 handler.go:92] blockchain not empty, fast sync disabled
I0314 17:39:23.439592    1198 cmd.go:114] Starting Geth/v1.3.5/darwin/go1.5.1
I0314 17:39:23.439692    1198 server.go:311] Starting Server
Fatal: Error starting Ethereum: listen udp :30303: bind: address already in use (possibly another instance of geth is using the same port)

**Comment by rfikki** [comment]
Looks like you might have more than one geth instance that is already running on the same port. You might check and kill that process.

**Comment by Wat3v4** [comment]
Any idea how to do it? 
Its weird, because i didn't do anything beside updating the software

**Comment by karalabe** [comment]
Try doing `ps aux | grep geth` on a terminal. What is the output?

**Comment by Wat3v4** [comment]
JZ    1745   0.0  0.0  2434840    776 s000  S+    6:16PM   0:00.00 grep geth

**Comment by karalabe** [comment]
Hmmm, this actually might be a similar issue to https://github.com/ethereum/go-ethereum/issues/1271. @fjl ^ ?

**Comment by karalabe** [comment]
@Wat3v4 What does `netstat -nl` print? Only the top section is interesting, the one beginning with `Active Internet connections (only servers)`?

**Comment by fjl** [comment]
I have previously had this issue when the OS X Firewall is on. It will hold onto the port even if the process is no longer running.

**Comment by Wat3v4** [comment]
@karalabe You want the whole table or first section? until "Active multipath internet connection" ?

@fjl How did you solve it? It just appeared.. my firewall was on even before that on older versions..

**Comment by karalabe** [comment]
Just that single section which has the open ports listed.

**Comment by Wat3v4** [comment]
Proto Recv-Q Send-Q  Local Address          Foreign Address        (state)    
tcp4       0      0  172.16.0.10.50901      81.161.59.86.80        SYN_SENT   
tcp4       0    613  172.16.0.10.50900      54.231.131.58.443      ESTABLISHED
tcp4       0    613  172.16.0.10.50898      54.231.131.58.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50897      54.231.131.58.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50896      54.231.131.58.443      ESTABLISHED
tcp4       0    613  172.16.0.10.50895      54.231.131.58.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50891      149.154.167.51.443     ESTABLISHED
tcp4      31      0  172.16.0.10.50889      192.30.252.124.443     CLOSE_WAIT 
tcp4      31      0  172.16.0.10.50888      192.30.252.128.443     CLOSE_WAIT 
tcp4       0      0  172.16.0.10.50887      192.30.252.88.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50886      52.0.212.134.443       ESTABLISHED
tcp4       0      0  172.16.0.10.50885      185.31.17.133.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50884      185.31.17.133.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50883      185.31.17.133.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50882      185.31.17.133.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50881      185.31.17.133.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50880      185.31.17.133.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50879      65.60.10.171.443       ESTABLISHED
tcp4      31      0  172.16.0.10.50878      192.30.252.128.443     CLOSE_WAIT 
tcp4       0      0  172.16.0.10.50875      54.230.45.55.443       ESTABLISHED
tcp4       0      0  172.16.0.10.50866      194.213.34.74.443      SYN_SENT   
tcp4       0      0  172.16.0.10.50848      54.231.163.26.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50824      216.58.214.206.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50812      192.30.252.87.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50786      216.58.214.206.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50700      149.154.167.51.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50646      81.161.59.93.80        ESTABLISHED
tcp4       0      0  172.16.0.10.50528      54.86.119.184.443      CLOSE_WAIT 
tcp4       0      0  172.16.0.10.50239      91.108.56.165.443      ESTABLISHED
tcp4       0      0  172.16.0.10.50238      149.154.167.91.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50237      149.154.167.91.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50236      149.154.167.91.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50231      149.154.167.91.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50227      149.154.175.100.443    ESTABLISHED
tcp4       0      0  172.16.0.10.50219      149.154.167.51.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50160      149.154.175.50.443     ESTABLISHED
tcp4       0      0  172.16.0.10.50087      13.80.123.255.443      ESTABLISHED
tcp4       0      0  172.16.0.10.49486      104.20.13.48.443       ESTABLISHED
tcp4       0      0  172.16.0.10.49344      162.125.32.129.443     ESTABLISHED
tcp4       0      0  172.16.0.10.49316      17.172.192.201.993     ESTABLISHED
tcp4       0      0  172.16.0.10.49254      91.190.218.61.12350    ESTABLISHED
tcp4       0      0  172.16.0.10.49246      17.172.192.201.993     ESTABLISHED
tcp4       0      0  172.16.0.10.49239      64.4.61.216.443        ESTABLISHED
tcp4       0      0  172.16.0.10.49237      213.199.179.164.40009  ESTABLISHED
tcp4       0      0  172.16.0.10.49228      74.125.136.109.993     ESTABLISHED
tcp4       0      0  172.16.0.10.49227      74.125.136.109.993     ESTABLISHED
tcp4       0      0  172.16.0.10.49187      17.172.238.201.5223    ESTABLISHED
tcp4       0      0  172.16.0.10.49174      17.143.160.35.5223     ESTABLISHED
tcp4       0      0  127.0.0.1.49153        127.0.0.1.1023         ESTABLISHED
tcp4       0      0  127.0.0.1.1023         127.0.0.1.49153        ESTABLISHED
udp46      0      0  *.*                    *.*                               
udp4       0      0  224.0.0.1.5350         *.*                               
udp46 104155      0  *.30303                *.*                               
udp4       0      0  *.17500                *.*                               
udp46      0      0  *.60871                *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  172.16.0.10.25175      *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  127.0.0.1.60870        *.*                               
udp4       0      0  *.52348                *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4     113      0  *.56126                *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp6       0      0  *.63957                                       *.*                                                      
udp4       0      0  *.63957                *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp6       0      0  fe80::e8c9:1aff:fe97:1879%awdl0.123           *.*                                                      
udp4       0      0  172.16.0.10.123        *.*                               
udp6       0      0  fe80::1240:f3ff:fef0:469a%en1.123             *.*                                                      
udp6       0      0  fe80::1%lo0.123                               *.*                                                      
udp4       0      0  127.0.0.1.123          *.*                               
udp6       0      0  ::1.123                                       *.*                                                      
udp6       0      0  *.123                                         *.*                                                      
udp4       0      0  *.123                  *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp46      0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp6       0      0  *.5353                                        *.*                                                      
udp4       0      0  *.5353                 *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.*                    *.*                               
udp4       0      0  *.137                  *.*                               
udp4       0      0  *.138                  *.*

**Comment by Wat3v4** [comment]
![screen shot 2016-03-14 at 6 34 08 pm](https://cloud.githubusercontent.com/assets/17217522/13749635/63e4eb46-ea13-11e5-8751-e5aecb49c899.png)

**Comment by karalabe** [comment]
Ah, that wasn't the one, there should be one which starts with:

```
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State      
tcp        0      0 127.0.1.1:53            0.0.0.0:*               LISTEN 
[...]
```

Note, the parenthesis **`(only servers)`**.

**Comment by Wat3v4** [comment]
Oh, There is nothing like that, only this one i sent..

**Comment by karalabe** [comment]
Hmmm, seems to be a Linux only feature then :( I guess we're down to @fjl 's idea.

**Comment by fjl** [comment]
@Wat3v4 the linked issue will be resolved for geth 1.4, which is due in a few weeks. Until then, you can try disabling your Firewall. Sorry for the inconvenience.

**Comment by Wat3v4** [comment]
Its ok, thanks to the both of you!
